node/service/src/lib.rs

```diff
@@ -861,10 +861,15 @@ pub fn new_dev(
 				commands_stream,
 				select_chain,
 				consensus_data_provider: None,
-				create_inherent_data_providers: move |_, _| async move {
+				//TODO I need a type annotation on the block type here. What type should I use?
+				create_inherent_data_providers: move |block, ()| async move {
```

Comment:
@bkchr This is related to https://github.com/paritytech/substrate/pull/8526

I'm not sure what type annotation I need here.

Reply:
This is the hash and not the block.